servers/curated-corpus-api/schema-shared.graphql

```diff
@@ -50,6 +64,73 @@ type CorpusItemAuthor {
   sortOrder: Int!
 }
 
+"""
+An entity containing Corpus Items.
+"""
+type Section {
+  """
+  An alternative primary key in UUID format.
+  """
+  externalId: ID!
+  """
+  The title of the Section displayed to the users.
+  """
+  title: String!
+  """
+  The GUID of the Scheduled Surface. Example: 'NEW_TAB_EN_US'.
+  """
+  scheduledSurfaceGuid: ID!
+  """
+  Controls the display order of Sections.
+  """
+  sort: Int
+  """
+  The source which created the Section.
+  """
+  createSource: ActivitySource!
+  """
+  Indicates whether or not a Section is available for display.
+  """
+  active: Boolean!
+  """
+  An array of active and in-active SectionItems in a Section.
+  This field returns an empty array when creating a new Section or updating a Section.
+  """
+  sectionItems: [SectionItem!]!
+  """
+  A Unix timestamp of when the Section was created.
+  """
+  createdAt: Int!
+  """
+  A Unix timestamp of when the Section was last updated.
+  """
+  updatedAt: Int!
```

Comment:
i'm not sure how i feel yet, but what do you think about introducing `createdAt` and `updatedAt` fields into the public graph? we don't have those currently on any public graph afaict - i think bc we didn't see a client need for that info. i'm not sure if it hurts being available though? i guess i'm mostly wondering at changing our pattern here on what entity data we expose publicly...

i think my reaction is to not change our pattern, but curious if you see a reason to do so.

Reply:
ah yeah for sure i haven't payed attention that we don't expose these fields on the public graph. i don't see harm in exposing them but i do like to stick to current pattern. I'll remove these from the shared graphql file & extend these in the admin graph only!